tests/auth_tests.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+
+import json
+
+from app import create_app
+
+class UserAuthTestCase(Unittest.TestCase):
+    '''User Authentication TestCase'''
+
+    def setUp(self):
+        """Set up test variables. """
+
+        self.app = create_app("testing")
+
+        '''initialize the test client'''
+        self.client = self.app.test_client
+
+        '''This is the user test json data with a predefined email and password'''
+        self.user_data = {
+            'email': 'user@testmail.com',
+            'password': 'pass123*'
+
+        }
+        def tearDown(self):
+            self.app_context.pop()
+
+    def test_user_registration(self):
+        '''test user registration works correctly'''
+
+
+        res = self.client.post('/auth/register', data=self.user_data)
+
+        '''get the results returned in json format'''
+        result = json.loads(res.data.decode()
```

Comment:
missing end bracket

Reply:
i noticed it when I ran the file 
let me fix it, I am gateful for the review